js/hotgraphicModel.js

```diff
@@ -10,10 +11,21 @@ export default class HotgraphicModel extends ItemsComponentModel {
 
   setUpItems() {
     super.setUpItems();
+    const id = this.get('_id');
+
+    this.getChildren().forEach((child, index) => {
 
-    this.getChildren().forEach((child) => {
       // Set _pin for the item if undefined
       if (!child.get('_pin')) child.set('_pin', false);
+
+      const tooltip = child.get('_tooltip');
+      if (!tooltip?._isEnabled) return;
+      tooltip._id = `hotgraphic-pin-${id}-${index}`;
+      tooltips.register({
+        _classes: [ 'hotgraphic__pin-tooltip' ],
+        ...tooltip
+      });
```

Comment:
Please add the item details to the register json, such that we can add a default tooltip of `{{title}}` later.
```suggestion
      tooltips.register({
        ...child.toJSON(),
        _classes: [ 'hotgraphic__pin-tooltip' ],
        ...tooltip
      });
```